src/styles/blocks/card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: $distanceBetweenCards;
+  margin-bottom: 50px;
```

Comment:
block can't have outer spacing, add margin-bottom to mix class

Reply:
and add justify-content to parent container, so you also won't need to add margin-right to every card